src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISFunctions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Microsoft.AspNetCore.HttpSys.Internal;
+using static Microsoft.AspNetCore.Server.IISIntegration.IISDeleagates;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration
+{
+    internal unsafe interface IISFunctions
```

Comment:
IIISFunctions? :-) Then DefaultIISFunctions can be IISFunctions.

Reply:
Nooooooooooooooo that sounds so bad.